lib/matplotlib/tests/test_rcparams.py

```diff
@@ -332,71 +332,60 @@ def generate_validator_testcases(valid):
                      ),
          'fail': (('aardvark', ValueError),
                   )
-         },
-         {'validator': validate_markevery,
```

Comment:
This part of the diff is just removing one space at the front of each line (note how the quote under 'validator' was previously not aligned with the quote under 'fail' just above).